obsreport/obsreport.go

```diff
@@ -36,25 +36,27 @@ var (
 	useNew    = true
 
 	okStatus = trace.Status{Code: trace.StatusCodeOK}
+
+	// useAlwaysSample is used in tests to force spans to be sampled.
+	useAlwaysSample bool
```

Comment:
In your test you can use this:

https://godoc.org/go.opencensus.io/trace#ApplyConfig and set the default sampler https://godoc.org/go.opencensus.io/trace#Config to AlwaysOn

Reply:
I didn't find a way to capture the original sampler but this much better than the private flag.